### ISSUE TITLE
String formatting causes "TypeError: must be real number, not str"

### ISSUE THREAD

**Issue by arturscheibler** [jd/tenacity#321]
On [this line](https://github.com/jd/tenacity/blob/master/tenacity/after.py#L39) an attempt is made to format a function name into a float with 3 decimal numbers.

I guess the correct code would be like this:
```
        sec = sec_format % retry_state.seconds_since_start
        logger.log(
            log_level,
            f"Finished call to '{_utils.get_callback_name(retry_state.fn)}' after {sec}(s), "
            f"this was the {_utils.to_ordinal(retry_state.attempt_number)} time calling it.",
        )
```

**Comment by henry0312** [comment]
@jd an ungent fix is required by v8.0.0

**Comment by jd** [comment]
Fixed by #317

**Comment by jd** [comment]
That's probably because I did the release manually, I forgot it was automatic.
